app/controllers/pages_controller.rb

```diff
@@ -8,5 +8,6 @@ def home
         brand.price_category == current_user.preference_price && brand.style == current_user.preference_style
       end
     end
+    @brands = Brand.all.sort
```

Comment:
Hey! All looks good, wondering if this is supposed to be Brand.eco instead of Brand.all??

Reply:
This variable is going to be used for the home page search dropdown autopopulation (in the next version of search :D). It is brand.all because naughty brands need to be searchable. FYI this variable is not actually being used in this version but will be in my next iteration next week @emilyunderhill

Reply:
Ah ofcourse! Makes sense, thanks!